src/components/FilterPanel.jsx

```diff
@@ -60,3 +81,7 @@ export default function FilterPanel() {
     </Box>
   );
 }
+
+FilterPanel.propTypes = {
+  closePanel: PropTypes.func.isRequired,
```

Comment:
nit: `onClose` might be a better name. Handlers tend to be named `on` some action. `closePanel` could be misinterpreted as a boolean.